modin/pandas/general.py

```diff
@@ -759,3 +759,15 @@ def get_names(obj):
         return list(names[0]) if is_list_like(names[0]) else [names[0]]
     else:
         return None
+
+
+@_inherit_docstrings(pandas.to_datetime, apilink="pandas.to_timedelta")
+@enable_logging
+def to_timedelta(arg, unit=None, errors="raise"):  # noqa: PR01, RT01, D200
+    """
+    Convert argument to timedelta.
+    """
+    if isinstance(arg, Series):
+        query_compiler = arg._query_compiler.to_timedelta(unit=unit, errors=errors)
+        return Series(query_compiler=query_compiler)
+    return pandas.to_timedelta(arg, unit=unit, errors=errors)
```

Comment:
this feels really weird - is there a reason why we should return a pandas object from this method?.. I would prefer to return Modin Series here at all cases

Reply:
Fixed to convert pandas Series results to modin Series